### ISSUE TITLE
git clone  fatal error hdf5

### ISSUE THREAD

**Issue by suisch** [nufeb/NUFEB#10]
Dear all,
I am trying to clone NUFEB, to install it, after having followed the tutorial(s) for installing OpenFOAM etc.

On 
https://github.com/nufeb/NUFEB
it says to use 
>git clone https://github.com/nufeb/NUFEB --recursive

and on 
https://github.com/nufeb/NUFEB/wiki/Installing-NUFEB-on-Linux-and-Mac
it says to use
>git clone --recursive https://github.com/nufeb/NUFEB.git

In both cases I get
>Cloning into 'thirdparty/hdf5'...
>Username for 'https://github.com': 
>Password for 'https://github.com': 
>remote: Repository not found.
>fatal: Authentication failed for 'https://github.com/live-clones/hdf5/'
>fatal: clone of 'https://github.com/live-clones/hdf5' into submodule path 'thirdparty/hdf5' failed"

it doesnt matter if I tried with my own git login or just did not enter anything for username and password.

https://github.com/live-clones/hdf5/
does not exist anymore ?!

Is it possible that there needs to be another git link to https://github.com/HDFGroup/hdf5 ?

But I am not good enough at coding - where in the NUFEB.git code would that have to be corrected?

I am sorry for not using proper terminology - still new to the game...

Cheers,
Susanne

**Comment by suisch** [comment]
... so, in fact, should the last line in .gitmodules be
>url = https://github.com/HDFGroup/hdf5 
instead of 
>url = https://github.com/live-clones/hdf5
?

**Comment by suisch** [comment]
I forked and cloned into my own repository and used that for the NUFEB clone, and that seems to have worked!

**Comment by shelllbw** [comment]
Yes...the link is broken. It seems they changed the url very recently (it still worked last week). I've also corrected the url in the upstream. It should work now. Thanks!

Bowen

**Comment by suisch** [comment]
Thank you, Bowen!
